apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/V1/SessionPoolTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright 2018 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.Grpc;
+using Google.Cloud.Spanner.Data;
+using Google.Cloud.Spanner.Data.IntegrationTests;
+using Google.Cloud.Spanner.V1.Internal.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Spanner.V1.PoolRewrite.IntegrationTests
+{
+    [Collection(nameof(AllTypesTableFixture))]
+    [CommonTestDiagnostics]
+    public class SessionPoolTests
```

Comment:
I suppose we could add lots more of tests here, actually probably as many as unit tests we have. But that can be done progressively anyways.

Reply:
I only really added this one to prove that refresh was working - at one point it was broken.

The session pool is used by just about everything, so it gets a pretty thorough testing.

Reply:
OK